### PR TITLE
Add Deployed ML models dashboard blog

### LINKED ISSUE #1409: [Blog] ML Commons Admin UI
**Describe the blog post your would like to write**
The ML-Commons admin UI is an intuitive and powerful interface for monitoring the responsing status of your machine learning models across various ML nodes. We are going to write this blog to discuss features and technical challenges.

**What is the title of the blog post?**
ML Commons Admin UI

**Who are the authors?**
wonglam, yych

**What is the proposed posting date?**
23th, March

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
Fixes #1409 

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2023-04-05-ml-deployment-status.md
Comment (pajuric) [inline _posts/2023-04-05-ml-deployment-status.md:10]: Please update the meta with the following changes: 
meta_keywords:  machine learning, ML Commons for OpenSearch, deployed models dashboard 
meta_description: Test the experimental deployed model dashboard functionality of OpenSearch Dashboards to view the responsiveness of ML nodes in your cluster.

**Review by wanglam — DISMISSED** [review]

File: _posts/2023-04-05-ml-deployment-status.md
Comment (wanglam) [inline _posts/2023-04-05-ml-deployment-status.md:14]: @Naarcha-AWS  As @pajuric said could we link to the opensearch 2.6 introduction blog? Here is the article link: https://opensearch.org/blog/introducing-opensearch-2-6/

Comment (Naarcha-AWS) [reply _posts/2023-04-05-ml-deployment-status.md:14]: Can do!

Comment (Naarcha-AWS) [reply _posts/2023-04-05-ml-deployment-status.md]: Done.

**Review by wanglam — DISMISSED** [review]

File: _posts/2023-04-05-ml-deployment-status.md
Comment (natebower) [inline _posts/2023-04-05-ml-deployment-status.md:45]: The image needs to be introduced in the text that precedes it.

**Review by natebower — CHANGES_REQUESTED** [review]
@Naarcha-AWS Please see my comments and changes and tag me when you are finished addressing them so that I can approve and push forward. Thanks!

File: _posts/2023-04-05-ml-deployment-status.md
Comment (natebower) [inline _posts/2023-04-05-ml-deployment-status.md:22]:
```suggestion
You can find the the deployed model dashboard in the **Machine Learning** section of OpenSearch Dashboards, as shown in the following image.
```

File: _posts/2023-04-05-ml-deployment-status.md
Comment (natebower) [inline _posts/2023-04-05-ml-deployment-status.md:48]:
```suggestion
- The model ID.
```

**Review by natebower — DISMISSED** [review]
Thanks @Naarcha-AWS! @pajuric This is ready to publish.

**Review by wanglam — DISMISSED** [review]

**Comment by pajuric** [comment]
@kolchfa-aws @Naarcha-AWS - Has this been finalized or are you waiting on editorial approval?

**Comment by natebower** [comment]
> @kolchfa-aws @Naarcha-AWS - Has this been finalized or are you waiting on editorial approval?

https://github.com/opensearch-project/project-website/pull/1496#pullrequestreview-1373297804

**Comment by kolchfa-aws** [comment]
@Naarcha-AWS I noticed you have one extraneous commit in this PR ([8641bf1](https://github.com/opensearch-project/project-website/pull/1496/commits/8641bf198b3cdfefb5dfbbef472bb2312e8204fc)).

File: _posts/2023-04-05-ml-deployment-status.md
Comment (pajuric) [inline _posts/2023-04-05-ml-deployment-status.md:8]: Publishing date will need to be updated. Please let me know the new scheduled date.

Comment (natebower) [reply _posts/2023-04-05-ml-deployment-status.md:8]: Publishing date is at the discretion of Marketing. Please update as appropriate.

**Review by dtaivpp — APPROVED** [review]
Content looks good. Going to see about getting DCO resolved

**Comment by dtaivpp** [comment]
Closing in light of #1523

**Pull request closed without merging**
